lib/tests/2wp.js

```diff
@@ -95,6 +95,58 @@ const execute = (description, getRskHost) => {
 
     });
 
+    it('should do a basic legacy pegin with the value above minimum', async () => {
+
+      // Arrange
+
+      const initialBridgeBalance = Number(await rskTxHelper.getBalance(BRIDGE_ADDRESS));
+      const initialBridgeUtxosBalance = await getBridgeUtxosBalance(rskTxHelper);
+      const initialFederationAddressBalanceInSatoshis = await getBtcAddressBalanceInSatoshis(btcTxHelper, federationAddress);
+      const senderRecipientInfo = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
+      const initialSenderAddressBalanceInSatoshis = await getBtcAddressBalanceInSatoshis(btcTxHelper, senderRecipientInfo.btcSenderAddressInfo.address);
+      // Value above minimum
+      const peginValueInSatoshis = minimumPeginValueInSatoshis + Number(btcToSatoshis(0.1));
+
+      // Act
+
+      const btcPeginTxHash = await sendPegin(rskTxHelper, btcTxHelper, senderRecipientInfo.btcSenderAddressInfo, satoshisToBtc(peginValueInSatoshis));
+      await ensurePeginIsRegistered(rskTxHelper, btcPeginTxHash);
+
+      // Assert
+
+      // The btc pegin tx is already marked as processed by the bridge
+      const isBtcTxHashAlreadyProcessed = await bridge.methods.isBtcTxHashAlreadyProcessed(btcPeginTxHash).call();
+      expect(isBtcTxHashAlreadyProcessed).to.be.true;
```

Comment:
Should this maybe be part of `ensurePeginIsRegistered`?

Reply:
It depends. Somethings we want to assert that this is false. Inside `ensurePeginIsRegistered` it should always be true. But we can put it inside the `ensurePeginIsRegistered` and when needed, assert from the test that it is false.